src/Sylius/Bundle/AdminBundle/Resources/config/app/config.yml

```diff
@@ -237,15 +239,23 @@ sylius_ui:
                         
         sylius.admin.layout.javascripts:
             blocks:
+                scripts:
+                    template: "@SyliusAdmin/Layout/_scripts.html.twig"
+                    priority: 10
                 legacy:
                     template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    priority: 5
                     context:
                         event: sylius.admin.layout.javascripts
                         
         sylius.admin.layout.stylesheets:
             blocks:
+                styles:
+                    template: "@SyliusAdmin/Layout/_styles.html.twig"
```

Comment:
Is the file name proper? You've deleted `{% include '@SyliusAdmin/_styles.html.twig' %}`

Reply:
No I did not delete it, but it has been mark as deprecated so I guess I have to set `@SyliusAdmin/_styles.html.twig` now and wait for deletion of the deprecated file ?